addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/HashDisclosureScanRuleUnitTest.java

```diff
@@ -112,16 +113,17 @@ void shouldRaiseAlertWhenResponseContainsOsxSha1AtLowThreshold() throws Exceptio
         scanHttpResponseReceive(msg);
         // Then
         assertThat(alertsRaised.size(), is(1));
-        assertThat(alertsRaised.get(0).getName(), is("Hash Disclosure - Mac OSX salted SHA-1"));
+        assertThat(alertsRaised.get(0).getName(), startsWith("Hash Disclosure - "));
         assertThat(alertsRaised.get(0).getEvidence(), is(hashVal));
+        assertThat(alertsRaised.get(0).getRisk(), is(equalTo(Alert.RISK_LOW)));
     }
 
     @ParameterizedTest
     @EnumSource(
             value = Plugin.AlertThreshold.class,
             names = {"HIGH", "MEDIUM"})
-    void shouldNotRaiseAlertWhenResponseContainsOsxSha1InJsAtNonLowThreshold(
-            AlertThreshold threshold) throws Exception {
+    void shouldNotRaiseAlertWhenResponseContainsHashInJsAtNonLowThreshold(AlertThreshold threshold)
```

Comment:
e.g. `shouldNotRaiseAlertWhenJavascriptResponseContainsHashAtNonLowThreshold` so the two methods have consistent naming.